styles/prosilver/template/event/viewtopic_body_footer_before.html

```diff
@@ -0,0 +1,6 @@
+<!-- IF not $INCLUDED_WOLFSBLVTADVANCEDPOLLSJS -->
+	<!-- INCLUDEJS js/functions.js -->
+	<!-- INCLUDEJS js/onload.js -->
+	<!-- INCLUDEJS js/scoring_topic.js -->
+    <!-- DEFINE $INCLUDED_WOLFSBLVTADVANCEDPOLLSJS = true -->
+<!-- ENDIF -->
```

Comment:
For me, it is more clean to add the js files very close to the place where they are needed. That is the whole purpose of the INCLUDEJS syntax.
That also makes sure JS is loaded everytime when the template is loaded.